.github/workflows/setup_rh_config/action.yaml

```diff
@@ -27,3 +27,4 @@ runs:
         echo "token: ${{ inputs.token }}" >> ~/.rh/config.yaml
         echo "username: ${{ inputs.username }}" >> ~/.rh/config.yaml
         echo "api_server_url: ${{ inputs.api_server_url }}" >> ~/.rh/config.yaml
+        echo "enable_observability: true" >> ~/.rh/config.yaml
```

Comment:
Make enable_observability -> disable_observability everywhere in the code, for constancy.